lib/common.sh

```diff
@@ -102,12 +102,7 @@ export BAREMETAL_OPERATOR_IMAGE=${BAREMETAL_OPERATOR_IMAGE:-"quay.io/metal3-io/b
 export OPENSTACK_CONFIG=$HOME/.config/openstack/clouds.yaml
 
 # CAPI version
-export V1ALPHA2_SWITCH=${V1ALPHA2_SWITCH:-"false"}
-if [ "${V1ALPHA2_SWITCH}" == true ]; then
-  export CAPI_VERSION=${CAPI_VERSION:-"v1alpha2"}
-else
-  export CAPI_VERSION=${CAPI_VERSION:-"v1alpha1"}
-fi
+export CAPI_VERSION=${CAPI_VERSION:-"v1alpha2"}
```

Comment:
the default should be still v1alpha1 no??

Reply:
yes, done